### PR TITLE
Rename methods

### PR DESCRIPTION
**Pull request by vinceatbluelabs**
This PR renames the following methods:

* can_unload_this_format -> can_unload_format
* can_move_to_this_format -> can_move_to_format

This matches can_load_format / can_move_from_format, renamed in https://github.com/bluelabsio/records-mover/pull/116

### CODE REVIEW

File: tests/unit/db/redshift/test_redshift_db_driver_format_negotiation.py [hunk 1]
```diff
@@ -33,10 +33,10 @@ def test_can_load_this_format(self):
             assert_called_with(mock_source_records_format)
         self.assertEqual(out, self.mock_redshift_loader.can_load_this_format.return_value)
 
-    def test_can_unload_this_format(self):
+    def test_can_unload_format(self):
```
Comment (crvena-sonja) [inline tests/unit/db/redshift/test_redshift_db_driver_format_negotiation.py:36]: To stick with the style of most test names, shouldn't this have `_true` at the end?

File: tests/unit/records/sources/test_fileobjs.py [hunk 2]
```diff
@@ -99,15 +99,15 @@ def test_known_supported_records_formats(self):
         out = source.known_supported_records_formats()
         self.assertEqual(out, [mock_records_format])
 
-    def test_can_move_to_this_format_yes(self):
+    def test_can_move_to_format_yes(self):
```
Comment (crvena-sonja) [inline tests/unit/records/sources/test_fileobjs.py:102]: Also, `yes` should be `true` here as well, I think?

File: tests/unit/records/sources/test_table.py [hunk 3]
```diff
@@ -108,11 +108,11 @@ def target_can_move_from_format(source_candidate):
             return (source_candidate == mock_common_format or
                     source_candidate == mock_target_format_1)
 
-        def source_can_move_to_this_format(target_candidate):
+        def source_can_move_to_format(target_candidate):
```
Comment (crvena-sonja) [inline tests/unit/records/sources/test_table.py:111]:
```suggestion
        def test_source_can_move_to_format_true(target_candidate):
```

File: tests/unit/records/sources/test_table.py [hunk 4]
```diff
@@ -145,10 +145,10 @@ def target_can_move_from_format(source_candidate):
                                         mock_target_format_1,
                                         mock_source_format_1]
 
-        def source_can_move_to_this_format(target_candidate):
+        def source_can_move_to_format(target_candidate):
```
Comment (crvena-sonja) [inline tests/unit/records/sources/test_table.py:148]:
```suggestion
        def test_source_can_move_to_format_true(target_candidate):
```

**Review by crvena-sonja — APPROVED** [review]
Add a few suggestions here to align test function names with the style *I think* I'm seeing in this code base. Feel free to take 'em or leave 'em.

File: tests/unit/db/redshift/test_redshift_db_driver_format_negotiation.py (hunk 1 above)
Comment (vinceatbluelabs) [inline tests/unit/db/redshift/test_redshift_db_driver_format_negotiation.py:36]:
```suggestion
    def test_can_unload_format_true(self):
```

File: tests/unit/records/sources/test_fileobjs.py (hunk 2 above)
Comment (vinceatbluelabs) [inline tests/unit/records/sources/test_fileobjs.py:102]:
```suggestion
    def test_can_move_to_format_true(self):
```

Comment (vinceatbluelabs) [reply tests/unit/records/sources/test_fileobjs.py]: ✔️

File: tests/unit/records/sources/test_table.py [hunk 5]
```diff
@@ -108,11 +108,11 @@ def target_can_move_from_format(source_candidate):
             return (source_candidate == mock_common_format or
                     source_candidate == mock_target_format_1)
 
-        def source_can_move_to_this_format(target_candidate):
+        def test_source_can_move_to_format_true(target_candidate):
             return (target_candidate == mock_common_format or
                     target_candidate == mock_source_format_1)
 
-        self.mock_loader.can_move_to_this_format = source_can_move_to_this_format
+        self.mock_loader.can_move_to_format = source_can_move_to_format
```
Comment (bluelabsbutler) [inline tests/unit/records/sources/test_table.py:115]: F821 undefined name 'source_can_move_to_format'

File: tests/unit/records/sources/test_table.py [hunk 6]
```diff
@@ -145,10 +145,10 @@ def target_can_move_from_format(source_candidate):
                                         mock_target_format_1,
                                         mock_source_format_1]
 
-        def source_can_move_to_this_format(target_candidate):
+        def test_source_can_move_to_format_true(target_candidate):
             return target_candidate in [mock_common_format, mock_source_format_1]
 
-        self.mock_driver.can_move_to_this_format = source_can_move_to_this_format
+        self.mock_driver.can_move_to_format = source_can_move_to_format
```
Comment (bluelabsbutler) [inline tests/unit/records/sources/test_table.py:151]: F821 undefined name 'source_can_move_to_format'

File: tests/unit/db/redshift/test_redshift_db_driver_format_negotiation.py (hunk 1 above)
Comment (vinceatbluelabs) [reply tests/unit/db/redshift/test_redshift_db_driver_format_negotiation.py]: ✔️

File: tests/unit/records/sources/test_table.py [hunk 7]
```diff
@@ -145,10 +145,10 @@ def target_can_move_from_format(source_candidate):
                                         mock_target_format_1,
                                         mock_source_format_1]
 
-        def source_can_move_to_this_format(target_candidate):
+        def test_source_can_move_to_format_true(target_candidate):
```
Comment (vinceatbluelabs) [inline tests/unit/records/sources/test_table.py:148]:
```suggestion
        def source_can_move_to_format(target_candidate):
```

File: tests/unit/records/sources/test_table.py [hunk 8]
```diff
@@ -108,11 +108,11 @@ def target_can_move_from_format(source_candidate):
             return (source_candidate == mock_common_format or
                     source_candidate == mock_target_format_1)
 
-        def source_can_move_to_this_format(target_candidate):
+        def test_source_can_move_to_format_true(target_candidate):
```
Comment (vinceatbluelabs) [inline tests/unit/records/sources/test_table.py:111]:
```suggestion
        def source_can_move_to_format(target_candidate):
```

**Pull request merged**